src/main/java/wayoftime/bloodmagic/common/item/ItemLivingArmor.java

```diff
@@ -44,14 +41,23 @@ public ItemLivingArmor(EquipmentSlot slot)
 	@Override
 	public String getArmorTexture(ItemStack stack, Entity entity, EquipmentSlot slot, String type)
 	{
+		String color = "red";
+
+		for (ItemStack itemStack : entity.getArmorSlots()){
```

Comment:
The chestplate would always be found in the second slot, no? Is this loop necessary?

Reply:
I haven't opened a Minecraft project in a while, but doesn't `getArmorSlots()` return either an array or a list? Slots don't just change position so you should just be able to get it by index.

Reply:
`getArmorSlots()` returns an `Iterable` not a list. Can't think of simple way to access the list directly